Documentation/Security/GuidelinesIntegrators/InstallTool.rst.txt

```diff
@@ -16,12 +16,14 @@ the Install Tool should be accessible. The second mechanism is a
 password protection, which is independent of all backend user
 passwords.
 
+The Install Tool can found as a stand alone application via ``https://<domain.tld>/typo3/install.php``.
+It also integrates with the backend, but is only available for logged in users with administrator privileges.
```

Comment:
I guess you need `System Maintainer` privileges since 9.x? As mentioned below. Maybe we can place a reference in order to not repeat information.